app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,84 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ *= require foundation_and_overrides
+
+ */
+
+ $yellow: #FFEEAC;
+ $blue: #141F3D;
+ $black: #060208;
+ $gold: #8A5A2C;
+ $brightred: #BE1214;
+ $gray: #8E8A86;
+
+ // blue color palette
+ $white: #F1F0F1;
+ $mid-level-blue: #011E56;
+ $dark-blue: #000F23;
```

Comment:
Color `#000F23` should be written as `#000f23`<br>Line should be indented 0 spaces, but was indented 1 spaces